db/migrate/20120903092956_devise_create_users.rb

```diff
@@ -34,7 +34,7 @@ def change
       # t.string :authentication_token
 
 
-      t.timestamps
+      t.timestamps null: true
```

Comment:
OK. Let's go with this for now, and raise an issue to disallow NULL values
for timestamps (may require data migrations).